public/main.js

```diff
@@ -42,103 +36,182 @@ function getTodoList(callback) {
     createRequest.send();
 }
 
+function createIconButton(id, icon) {
+    var Button = document.createElement("button");
+    Button.setAttribute("type", "button");
+    Button.setAttribute("class", "btn btn-default");
+    Button.setAttribute("id", id);
+    var span = document.createElement("span");
+    span.setAttribute("class", icon);
+    span.setAttribute("aria-hidden", "true");
+    Button.appendChild(span);
+    return Button;
+}
+
+function createReq(method, url, body, errorMsg, callback){
+    fetch( url, {
+        method: method,
+        headers: {
+            "Content-type": "application/json"
+        },
+        body: body
+    })
+    . then(function(res) {
+        callback();
+    })
+    .catch(function(res){
+        error.textContent = errorMsg + " Server returned " + res.status + " - " + res.responseText;
+    });
+}
 
 
 function reloadTodoList() {
     while (todoList.firstChild) {
         todoList.removeChild(todoList.firstChild);
     }
     todoListPlaceholder.style.display = "block";
+
+    var totalItems = 0;
```

Comment:
What is the correct scope for these 2? Can they be moved into the function below?